kubernetes-client/src/main/java/io/fabric8/kubernetes/client/DefaultKubernetesClient.java

```diff
@@ -316,4 +318,9 @@ public SubjectAccessReviewDSL subjectAccessReviewAuth() {
     return new SubjectAccessReviewDSLImpl(httpClient, getConfiguration());
   }
 
+  @Override
+  public SharedInformerFactory informers() {
```

Comment:
It would be nice to have a `SharedInformerFactory informers(ExecutorService executor)` method for managed environments which the ForkJoinPool would not be used

Reply:
Thanks for your suggestion, I'll add it.